components/collective-page/graphql/queries.js

```diff
@@ -134,7 +134,7 @@ export const getCollectivePageQuery = gql`
           }
         }
       }
-      subCollectives: members(role: "SUB_COLLECTIVE") {
+      relatedCollectives: members(role: "RELATED_COLLECTIVE") {
```

Comment:
In the DB the role is still `SUB_COLLECTIVE`, so if we merge that as-is it will break the feature. We need a matching API PR to make that work.

Reply:
Thanks. I've done the migration PR; https://github.com/opencollective/opencollective-api/pull/3728